manager/manifests/istio.yaml

```diff
@@ -0,0 +1,605 @@
+# Copyright 2019 Cortex Labs, Inc.
```

Comment:
Is it possible to just keep any modifications you made to this file?

Reply:
yup, cleaned up cofig